src/silx/app/view/main.py

```diff
@@ -71,6 +73,39 @@ def createParser():
     return parser
 
 
+def filesArgToUrls(filenames: Iterable[str]) -> Generator[object, None, None]:
+    """Expand filenames and HDF5 data path in files input argument"""
+    import silx.io
+    from silx.io.utils import match
+    from silx.io.url import DataUrl
+    import silx.utils.files
```

Comment:
might be moved to top of the file

Reply:
Some might, but anything importing `h5py` should be imported after `HDF5_USE_FILE_LOCKING` is set from the command line arguments.
But it's definitely worth a comment!